shared/src/main/scala/pl/metastack/metatime/Scheduler.scala

```diff
@@ -20,6 +20,27 @@ trait Scheduler {
     })
 
   def currentTimeMillis(): Long = System.currentTimeMillis()
+
+  def at(time: Time)(f: => Unit): Cancelable =
+    at(time.fromNow())(f)
+  def at(dateTime: DateTime)(f: => Unit): Cancelable =
+    at(dateTime.fromNow())(f)
+  def at(time: Offset[_])(f: => Unit): Cancelable =
+    scheduleOnce(Math.abs(time.component.asInstanceOf[Component].milliseconds()).millis)(f)
```

Comment:
Same error is received when line 28 is changed to following:
`def at[T <: Component](time: Offset[T])`